src/utils/azure.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from '../localize';
+
+// tslint:disable-next-line:export-name
+export function getResourceGroupFromId(id: string): string {
+    const matches: RegExpMatchArray | null = id.match(/\/subscriptions\/(.*)\/resourceGroups\/(.*)\/providers\/(.*)\/(.*)/);
```

Comment:
Suggestion:

   [ , , resourceGroup ] = id.match(...

Reply:
Didn't know you could do that. That's nifty, but I get a type error when I do that because 'id.match' can return null

---
In reply to: [147806237](https://github.com/Microsoft/vscode-azurefunctions/pull/32#discussion_r147806237) [](ancestors = 147806237)